docs/configuration.md

```diff
@@ -1931,8 +1931,8 @@ editing the `conf` file in a text editor. Use the examples as reference.
     </tr>
     <tr>
         <td>Default</td>
-        <td colspan="2">Automatic.
-            Sunshine will use the first capture method available in the order of the table above.</td>
+        <td colspan="2">Automatic<br>
```

Comment:
Please revert. We will not change the default capture method until this has been stress tested.